spec/sinatra_logging_in_and_out_spec.rb

```diff
@@ -1,49 +1,60 @@
 require 'spec_helper'
 
-describe 'App' do
+describe 'ApplicationController' do
   describe "GET '/'" do
     it "returns a 200 status code" do
```

Comment:
Hmm you aren't testing that the site looks different if you are logged in or not logged in. Test for that. This is a great time to use `context`s